index.bs

```diff
@@ -8292,15 +8292,19 @@ The [=remote end steps=] given |session| and |command parameters| are:
 1. Let |arguments| be the <code>arguments</code> field of |command
    parameters| if present, or an empty [=/list=] otherwise.
 
-1. Let the |contexts| be the value of the <code>contexts</code> field of |command parameters| if it is present or null if it isn’t.
+1. If the <code>contexts</code> field of |command parameters| is present:
 
-1. If |contexts| is not null:
+   1. Let |contexts| be an empty [=/set=].
```

Comment:
I'm not sure if the declaration can be inside "if" condition.

Reply:
Not sure either, maybe a better thing would be `Let contexts be null` and then set it to a Set in the if statement.

Reply:
I don't think this is important though )